controllers/cache_handler.go

```diff
@@ -118,7 +118,8 @@ func (h DisruptionTargetWatcherHandler) OnChangeHandleNotifierSink(oldPod, newPo
 			continue
 		}
 
-		eventType := corev1.EventTypeWarning
+		eventType := chaosv1beta1.Events[eventReason].Type
+
 		if eventReason == chaosv1beta1.EventNodeRecoveredState || eventReason == chaosv1beta1.EventPodRecoveredState {
```

Comment:
We may not need this any longer as their default event type seems to be `EventTypeNormal` for both of them

🎉

Reply:
Good catch! 👏